queries.sql

```diff
@@ -8,3 +8,41 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon'; 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
```

Comment:
- [ OPTIONAL ] I think here it would be better if you add date_of_birth >= '2022-01-01' because the query is `DELETE` all animals born after Jan 1st, 2022, so the begining of the date also will be counted with it 🔥🙏🏻

Reply:
@Reem-lab Thank you for the comment I will work on it.